manifests/init.pp

```diff
@@ -176,7 +176,7 @@
       owner  => 'root',
       group  => 'root',
       mode   => '0644',
-      source => "puppet:///modules/epel/RPM-GPG-KEY-EPEL-${os_maj_release}",
+      content => template("epel/RPM-GPG-KEY-EPEL-${os_maj_release}.erb"),
```

Comment:
Given the expectation of what a template file is, I think it's worth adding a comment such as `# Use a template to avoid a dependency on a puppet master; nothing is parsed in the template` to ensure there is no confusion.

Reply:
This could have used `content` with the `file` function instead.  The file content is still in the catalog this way, but we avoid unnecessary processing of erb.  I've done this in https://github.com/voxpupuli/puppet-epel/pull/113